components/raftstore/src/store/async_io/write.rs

```diff
@@ -864,7 +864,7 @@ where
     ER: RaftEngine,
 {
     /// Mailboxes for sending raft messages to async ios.
-    writers: Arc<VersionTrack<SenderVec<EK, ER>>>,
+    writers: Arc<Mutex<SenderVec<EK, ER>>>,
```

Comment:
There's no need to use a lock, And you don't need to make it clonable so you can drop the `Arc`.

Reply:
Arc is needed in the following code:
```
pub fn senders(&self) -> WriteSenders<EK, ER> {
        WriteSenders::new(self.writers.clone())
    }
```